client/src/modules/stock/inventories/modals/search.modal.js

```diff
@@ -60,10 +73,27 @@ function SearchInventoriesModalController(data, Notify, Instance, Store, Periods
   vm.cancel = function cancel() { Instance.close(); };
 
   vm.submit = function submit() {
+    // Set the label of status Value
+    if (vm.searchQueries.status) {
```

Comment:
I can imagine that other developers will want to use this too.

Why don't you make an object map and put it in a service (like the `StockService`)?  That way, you could do this:
```js
// this is in a stock service
service.statusLabelMap = {
  'sold_out' : 'STOCK.STATUS.SOLD_OUT',
  // ... etc ...
};

// this is in your submit() function.
if (vm.searchQueries.status) {
  displayValues.status = service.statusLabelMap[vm.searchQueries.status];
}
```
That way, if multiple controllers need to have the status --> i18n key mapping, they can just import your service!

Reply:
Sorry for the few Comment, some times when I update my branch after resolving a conflict, I lose the message of my former Commit